build-machinery/make/targets/openshift/codegen.mk

```diff
@@ -0,0 +1,34 @@
+CODEGEN_PKG ?=./vendor/k8s.io/code-generator/
+CODEGEN_GENERATORS ?=all
+CODEGEN_OUTPUT_BASE ?=../../..
+CODEGEN_GO_HEADER_FILE ?=/dev/null
+
+#ifneq ($(and $(CODEGEN_GENERATORS),$(CODEGEN_OUTPUT_PACKAGE),$(CODEGEN_API_PACKAGE),$(CODEGEN_GROUPS_VERSION)),)
```

Comment:
why commented out?

Reply:
leftover, thanks for catching that. I have tried this pattern but it didn't work out in all the places (especially in combination with eval). And make has `--warn-undefined-variables` to catch it so I let it go in the interest of time.